src/middlewared/middlewared/api/v25_04_0/acme_dns_authenticator.py

```diff
@@ -54,6 +54,16 @@ class CloudFlareSchemaArgs(CloudFlareSchema):
     pass
 
 
+class DigitalOceanSchema(BaseModel):
+    authenticator: Literal['digitalocean']
+    api_token: Secret[NonEmptyString | None] = Field(default=None, description='API Token')
```

Comment:
Why do we allow a null api token ?

Reply:
@sonicaj good catch :+1: it's fixed in the latest push (I missed it's another case above where I copied it from)